app/pages/Login.jsx

```diff
@@ -17,7 +17,7 @@ class Login extends Component {
                     <LanguagePicker />
                     <EnterInput placeholder={this.props.strings.namePlaceholder} icon={icons.people} onEnter={this.props.onLogin} ref="input" />
                     <Button label={this.props.strings.buttonLabel} accent raised onClick={() => {
-                        const text = this.refs.input.value();
+                        const text = this.refs.input.state.value;
```

Comment:
Thanks, I indeed removed value() from the component. Really what I should do is add an onChange on EnterInput, and keep the value a state prop in Login.jsx (updated on each onChange).